amlb/runners/docker.py

```diff
@@ -54,7 +57,7 @@ def _start_container(self, script_params=""):
         inst_name = f"{self.sid}.{str_sanitize(str_digest(script_params))}"
         cmd = (
             "docker run --name {name} {options} "
-            '-u "$(id -u $USER):$(id -g $USER)" '
+            f'-u "{self.userid}:{self.usergid}" '
```

Comment:
for Windows users, docker mode is often the only mode available to run most of the frameworks, so we need to ensure that this still works on Win.

@PGijsbers I now wonder how the original `$(id -u $USER)` calls worked on Win, are we requiring Bash on Windows? In this case, how does the Py commands `os.getuid()` and `os.getgid()` behave on Win?

Reply:
I can't test that, but Windows (and Mac), don't mess up root with user, i.e., you run a docker a root and save files, those files will belong to original windows/mac user and not to the root, like it happens in Linux.
Perhaps, just wrap these lines with a `if linux` ...

Reply:
Seems reasonable to me